hrm-service/src/contact/contact-data-access.ts

```diff
@@ -56,7 +56,7 @@ type QueryParams = {
   dateFrom?: string;
   contactNumber?: string;
   helpline?: string;
-  onlyDataContact: boolean;
+  onlyDataContacts: boolean;
```

Comment:
Question: Was this missing `s` the cause of the issue? Or you're just refactoring it to give a better name?